client/controllers/article.coffee

```diff
@@ -1,4 +1,9 @@
 Template.article.events
   "click .delete-article": (e) ->
-    if window.confirm("Are you sure you want to delete this article?\nAll locations mentioned only in this article will be deleted as well.")
+    incidentCount = Incidents.find({url: @url}).count()
```

Comment:
Could you change the query to only count the incidents for the current event. The subscription is probably only returning incidents for the current event, but that could change.